### DISCUSSION TITLE
Setting up jumpcloud

### DISCUSSION THREAD

**Discussion by rodude123** [9p4/jellyfin-plugin-sso#149] [Q&A] [answered]
I'm setting up Jumpcloud with Jellyfin; where do I edit the SAML config? I can see the UI page has not been added yet, so how do I do it then?

**Comment by 9p4** [comment, marked as answer]
There is no UI for SAML yet. You have to configure it through the API. Please refer to the README for more information on how to configure SAML.

**Comment by rodude123** [comment]
Ok that's fine, I will use the API then

**Comment by rodude123** [comment]
just quick question on setting this up, jump cloud uses something called an ACS URL what's the ACS URL for this, is it the 
https://myjellyfin.example.com/sso/SAML/start/PROVIDER_NAME

**Comment by rodude123** [comment]
ok, so I have managed to set it up, but how do I connect the already added user to the one in SSO, as they both have the same username? I'd like my account to be admin and sso compatible too

**Reply by 9p4** [reply]
You can use the linking view `/sso/Linking`

**Comment by rodude123** [comment]
So I noticed the permissions get reverted, so if I set a user to be allowed to manage the server, it saves until I log out. Then, when I log in, it's back to normal.

**Reply by 9p4** [reply]
Yes, the plugin sets the permissions based on the SAML permissions.

**Reply by rodude123** [reply]
how do I do that? any ideas

**Reply by 9p4** [reply]
How do you do what?

**Reply by rodude123** [reply]
I mean how do I set correct permissions so that when said user logs in they are automatically allowed to admin the server

**Reply by 9p4** [reply]
You can enable plugin-based-authorization and add your users to the "admin" roles.

**Reply by rodude123** [reply]
So enable this `enableAuthorization` then how does jumpcloud handle group based permisions?

**Reply by 9p4** [reply]
I don't use jumpcloud so I can't really help with that side of things. A quick glance shows https://jumpcloud.com/support/get-started-applications-saml-sso#managing-employee-access-to-applications

**Reply by rodude123** [reply]
I want all users to have access to jellyfin but I want myself to have admin access so not allowing them to use jellyfin would not be an option. Is there a way to do group based perms

**Reply by rodude123** [reply]
alternatively could i disable `enableAuthorization` and handle it manually

**Reply by 9p4** [reply]
I don't think that feature is there yet. See also: https://github.com/9p4/jellyfin-plugin-sso/issues/147 and https://github.com/9p4/jellyfin-plugin-sso/issues/111

**Reply by rodude123** [reply]
so setting `enableAuthorization` to false allowed me to handle permissions manually, which worked.
